code/modules/mob/living/simple_animal/friendly/dog.dm

```diff
@@ -418,6 +418,42 @@
 					setDir(i)
 					sleep(1)
 
+/mob/living/simple_animal/pet/dog/corgi/Ian/narsie_act()
+	playsound(src, 'sound/magic/demon_dies.ogg', 75, TRUE)
+	var/mob/living/simple_animal/pet/dog/corgi/narsie/N = new(get_turf(src))
```

Comment:
use loc

Reply:
In what universe do we prefer `loc` over `get_turf()`?